src/lib.rs

```diff
@@ -2448,6 +2451,18 @@ impl Connection {
             self.streams.collect(stream_id, local);
         }
 
+        qlog_with!(self.qlog_streamer, q, {
```

Comment:
Actually, wait, I just realized that this is in the transport code. I don't think we should be logging HTTP/3 events from here...

Reply:
Ah nvm, I read the PR description now.